air-interpreter/src/marine.rs

```diff
@@ -63,7 +65,12 @@ pub fn invoke_tracing(
     tracing_params: String,
     tracing_output_mode: u8,
 ) -> InterpreterOutcome {
-    logger::init_tracing(tracing_params, tracing_output_mode);
+    if unsafe { !TRACING_IS_INITED } {
```

Comment:
I've looked at the tracing_subscriber docs.  It seems we can use two alternative approaches:
1. `init_tracing` should use `try_init`.  However, it returns opaque error object, and we cannot tell if it is existing object or some other error.
2. Run `execute_air` inside `with_default(...)`.  It seems to be perfect solution as it allows different tracing options on each invocation.

See https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/fn.fmt.html for examples for both options.